CONTRIBUTING.md

```diff
@@ -31,7 +31,28 @@ A maintainer will request changes if necessary. For content contributions, don't
 
 ## Setup Instructions
 
-This project is built using [Hugo](https://gohugo.io/), an open-source static site generator. To get the project up and running:
+This project is built using 
+- [Hugo](https://gohugo.io/), an open-source static site generator.
+- [Docker](https://www.docker.com/), a platform to generate containerized environments.
+
+Using Docker for the project is preferred to minimize conflicts arising from machine, OS, or dependency variations.
+
+### To get the project up and running Docker:
+1. Download and Install [Docker](https://docs.docker.com/get-docker/)
+2. Run `docker compose up`
+3. Visit the site at http://localhost:1313
+4. Stop the container with `Ctrl + C` or `docker compose down`
```

Comment:
I would swap the "or" for an "and" here since it is slightly different from what `Ctrl + C` does

Reply:
Would it be more accurate to say `docker compose stop`

Reply:
See new proposal stop the container with either `Ctrl + C` or `docker compose stop`

Added additional bullet for clarity